src/current/_includes/releases/v23.2/v23.2.0-beta.3.md

```diff
@@ -0,0 +1,37 @@
+## v23.2.0-beta.3
+
+Release Date: December 13, 2023
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v23-2-0-beta-3-sql-language-changes">SQL language changes</h3>
+
+- Added the `sql.ttl.default_select_rate_limit` [cluster setting](../v23.2/cluster-settings.html) and the `ttl_select_rate_limit` [table storage parameter](../v23.2/with-storage-parameter.html#table-parameters) to set the TTL select rate limit. This sets the number of records per table per second per node that can be selected by the TTL job. [#115802][#115802]
+
+<h3 id="v23-2-0-beta-3-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug that could result in an incorrect `too few columns` error for queries that use `ANY <array>` syntax with a subquery. [#115592][#115592]
+- Fixed a bug that could cause `too few columns`/`too many columns` errors for queries that used `IN` or `NOT IN` with a non-trivial right operand, such as a subquery (rather than a constant tuple). [#115592][#115592]
+- Fixed a bug where [`CREATE INDEX`](../v23.2/create-index.html) with expressions could fail on materialized [views](../v23.2/views.html) when the declarative schema changer was used. [#115522][#115522]
+- Fixed a bug that could cause PL/pgSQL routines with `SELECT INTO` syntax to return early. This bug existed only in pre-release versions v23.2.0-beta.1 and v23.2.0-beta.2. [#115676][#115676]
+- Previously, in rare cases, CockroachDB could incorrectly evaluate queries with lookup [joins](../v23.2/joins.html) where `equality cols are key` when looking up into multiple ranges. This could either manifest as a stuck query or result in incorrect output. The bug was introduced in v22.2 and is now fixed. [#115580][#115580]
+- Fixed a durability bug in Raft log storage that was caused by incorrect syncing of filesystem metadata. It was possible to lose writes of a particular kind (`AddSSTable`) used by (e.g.) `RESTORE`. This loss was possible only under power-off or OS crash conditions. As a result, CockroachDB could enter a crash loop on restart. In the worst case of a coordinated power-off/crash across multiple nodes, this could lead to an unrecoverable loss of quorum. [#115841][#115841]
+- Fixed a bug where large jobs running with execution locality could result in the [gateway node](../v23.2/architecture/sql-layer.html#gateway-node) being assigned most of the work causing performance degradation and cluster instability. [#115876][#115876]
+- Fixed a bug that could cause side effects to happen out of order for PL/pgSQL routines in rare cases. This bug existed only in v23.2 alpha and beta pre-release versions. [#115840][#115840]
```

Comment:
nit: We could move this right after the other PL/pgSQL issue

Reply:
Done!